src/store/DevfileRegistries/index.ts

```diff
@@ -173,7 +173,7 @@ export const actionCreators: ActionCreators = {
 
         const schemav200 = await WorkspaceClient.restApiClient.getDevfileSchema('2.0.0');
         const schemav210 = await WorkspaceClient.restApiClient.getDevfileSchema('2.1.0');
-        const schemav220alpha = await WorkspaceClient.restApiClient.getDevfileSchema('2.2.0-alpha');
+        const schemav220alpha = await WorkspaceClient.restApiClient.getDevfileSchema('2.2.0');
```

Comment:
Why shouldn't you rename the variable as well?

Reply:
to make it clearer that it's not released yet and the latest stable to 210